src/model/transaction/applyEntityToContentBlock.ts

```diff
@@ -40,3 +40,16 @@ export default function applyEntityToContentBlock(
       }
     : contentBlock;
 }
+
+export function applyEntityToMutableCharacterList(
+  characterList: CharacterMetadata[],
+  start: number,
+  end: number,
+  entityKey: string | null,
+): void {
+  for (let i = start; i < end; i++) {
+    if (characterList[i].entity !== entityKey) {
```

Comment:
what does this line mean? can there not be multiple entities?

Reply:
Yes, each character has only a single entity. The entity itself can encompass many properties (BC's `ModelData`)